tests/image/test_build.py

```diff
@@ -27,6 +31,31 @@ def test_cli(docker_client_mock, project_dir, test_mock):
     test_mock.assert_called_with("dev", None)
 
 
+def test_cli_with_progress(docker_client_mock, project_dir, test_mock):
+    docker_client_mock.build.return_value = [
+        {"stream": "Step 1/3 : FROM some_image"},
+        {"stream": "some internal actions"},
+        {"stream": "Step 2/3 : RUN cmd1"},
+        {"stream": "some other actions"},
+        {"stream": "Step 3/3 : RUN cmd2"},
+        {"stream": "Successfully built 12345"}
+    ]
+    runner = CliRunner()
+    result = runner.invoke(cli, ["dev"])
+    assert result.exit_code == 0
+    placeholder = '█' if _supports_unicode(sys.stdout) else '#'
```

Comment:
Good question, I'm on it (have just fixed tests for python 2 and broken for python 3) 😄